experimental/Rings/QabAndPChars.jl

```diff
@@ -518,197 +517,157 @@ function quadratic_irrationality_info(a::QabModule.QabElem)
     return (x, y, m)
 end
 
+function isroot_of_unity(a::QabElem)
+  return istorsion_unit(a.data)
+  #=
+  b = a^a.c
+  return b.data == 1 || b.data == -1
+  =#
+end
+
+function Oscar.order(a::QabElem)
```

Comment:
Could you add a comment on what this is doing? What is the order of an element?

Reply:
I will add a comment, but it was already there. This is a first implementation of the abelian closure of Q that Claus did and it was just done to provide what the algorithms for binomial ideals needed.

Reply:
Did you add the comment?